EmpleoDotNet/Views/Home/Index.cshtml

```diff
@@ -16,7 +16,7 @@
                 <div class="jobs">
                     @foreach (var jobOpportunity in Model)
                     {
-                        <a href="@Url.Action("Detail", "JobOpportunity", new {id = jobOpportunity.Id})" class="featured" target="_blank">
```

Comment:
Pensé que era un error. No es muy normal que cuando le des a una propuesta te la abra en otra pantalla. No other site does that as far as i know. Es como que en amazon el te abra todos los links en una pantalla nueva... it's quite annoying imo.